lua/telescope/config/resolve.lua

```diff
@@ -139,7 +139,7 @@ end] = function(_, val)
 end
 
 _resolve_map[function(val)
-  return type(val) == "table" and val[1] >= 0 and val[1] < 1 and val["max"] ~= nil
+  return type(val) == "table" and #val > 0 and val[1] >= 0 and val[1] < 1 and val["max"] ~= nil
```

Comment:
There is already a PR for this: https://github.com/nvim-telescope/telescope.nvim/pull/2097

I prefer to merge a new contribution, also because it was opened prior to your fix :laughing: 

So you can drop your commit and then rebase

Reply:
yayaya will do